src/terminology.md

```diff
@@ -8,38 +8,54 @@ We group terms into related categories as follows:
 
 ## Protocol Concepts
 
-- *Assured Finality*: A protocol property that guarantees that final transactions cannot be reverted by that protocol. As with all protocol guarantees, a protocol assumes certain conditions must be met.
+<span id="definition-assured-finality"></span>**Assured Finality**: A protocol property that guarantees that final transactions cannot be reverted by that protocol. As with all protocol guarantees, a protocol assumes certain conditions must be met.
```

Comment:
I would use
``html
<span id="definition-assured-finality">**Assured Finality**: A protocol property that guarantees that final transactions cannot be reverted by that protocol. As with all protocol guarantees, a protocol assumes certain conditions must be met.</span>
```
We can fix that up for all definitions separately.